<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="test-PC" name="test1.Oc" tests="10" failures="1" timestamp="13 Mar 2017 10:11:08 GMT" time="32.399" errors="0">
  <testcase name="check" time="4.502" classname="test1.Oc"/>
  <testcase name="NCIIC_Pic" time="2.115" classname="test1.Oc"/>
  <testcase name="NCIIC_Check" time="2.178" classname="test1.Oc"/>
  <testcase name="login" time="9.968" classname="test1.Oc"/>
  <testcase name="xuexin" time="2.140" classname="test1.Oc"/>
  <testcase name="Part4" time="0.000" classname="test1.Oc">
    <failure type="org.testng.TestNGException" message="
Parameter &#039;OfficeTelephone1&#039; is required by @Test on method Part4 but has not been marked @Optional or defined
in F:\test\workspace\testng\src\Score-all0.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'OfficeTelephone1' is required by @Test on method Part4 but has not been marked @Optional or defined
in F:\test\workspace\testng\src\Score-all0.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:155)
	at org.testng.internal.Parameters.createParameters(Parameters.java:358)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:450)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- Part4 -->
  <testcase name="Part2" time="4.671" classname="test1.Oc"/>
  <testcase name="Sell_Pic" time="2.109" classname="test1.Oc"/>
  <testcase name="SheBao" time="2.189" classname="test1.Oc"/>
  <testcase name="Part1" time="2.527" classname="test1.Oc"/>
</testsuite> <!-- test1.Oc -->
